pkgs/development/python-modules/sphinxcontrib-openapi/default.nix

```diff
@@ -1,26 +1,55 @@
 { lib
 , buildPythonPackage
-, fetchPypi
+, deepmerge
+, fetchFromGitHub
+, fetchpatch
 , isPy27
 , setuptools-scm
-, m2r
-, pyyaml
 , jsonschema
+, picobox
+, pyyaml
+, sphinx_mdinclude
 , sphinxcontrib_httpdomain
 }:
 
 buildPythonPackage rec {
   pname = "sphinxcontrib-openapi";
-  version = "0.7.0";
+  version = "unstable-2022-08-11";
   disabled = isPy27;
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "1c1bd10d7653912c59a42f727c62cbb7b75f7905ddd9ccc477ebfd1bc69f0cf3";
+  # Switch to Cilums' fork of openapi, which uses m2r instead of mdinclude,
+  # as m2r is unmaintained and incompatible with the version of mistune.
+  # See
+  # https://github.com/cilium/cilium/commit/b9862461568dd41d4dc8924711d4cc363907270b and
+  # https://github.com/cilium/openapi/commit/cd829a05caebd90b31e325d4c9c2714b459d135f
+  # for details.
+  src = fetchFromGitHub {
+    owner = "cilium";
+    repo = "openapi";
+    rev = "a1d4fca2e7c3ae3cca69593baade1ebc297a12ff";
+    hash = "sha256-4xW4Qgr7IR055B74M/zn9cdnTmC4Pg7QZB3mH4YjDk4=";
   };
 
+  patches = [
+    (
+      fetchpatch {
+        url = "https://github.com/cilium/openapi/pull/1/commits/184a41c90129397c07b1f69bd1a51ea3b94422d3.patch";
```

Comment:
Isn't that patch missing a try catch?

Reply:
I don't understand the question. What would I try/catch here?